ionic-team/ionic-docs
# fix(playground): update template generator to support Angular standalone

Updates the template generator to support Angular standalone.

How to test these changes:

1. Navigate to the `ionic-docs` repository
1. Run `npm run playground:new`
1. Select the following options when prompted:

    ```bash
    ? Is this playground for a component? › yes
    ? Which component is this playground for? › ion-button
    ? What should the playground path be? › withcss
    ? Select the Ionic Framework version for the playground › 8
    ? Generate custom CSS files? › yes
    ```

1. Run `npm run playground:new` again, but this time select the following options:

    ```bash
    ? Is this playground for a component? › yes
    ? Which component is this playground for? › ion-button
    ? What should the playground path be? › withoutcss
    ? Select the Ionic Framework version for the playground › 8
    ? Generate custom CSS files? › no
    ```

1. Add the following to `docs/api/button.md` before the `## Basic Usage` section:

    ```md
    ## Withcss

    import Withcss from '@site/static/usage/v8/button/withcss/index.md';

    <Withcss />

    ## Withoutcss

    import Withoutcss from '@site/static/usage/v8/button/withoutcss/index.md';

    <Withoutcss />
    ```

1. Start the docs server: `npm run start`
1. Open your browser and navigate to `http://localhost:3000/docs/api/button`
1. Verify the following:
    - The Angular code examples look correct
    - The StackBlitz previews open without issues
